test/module.test.ts

```diff
@@ -232,6 +232,66 @@ describe('ssr', async () => {
     })
   })
 
+  it('detects cloudfront headers - mobile', async () => {
+    const html = await $fetch('/', {
+      headers: {
+        'User-Agent': 'Amazon CloudFront',
+        'Cloudfront-Is-Mobile-Viewer': 'true'
+      }
+    })
+    const { isMobile, isMobileOrTablet } = parseHtml(html)
+
+    expect({ isMobile, isMobileOrTablet }).toEqual({ isMobile: true, isMobileOrTablet: true })
+  })
+
+  it('detects cloudfront headers - tablet', async () => {
+    const html = await $fetch('/', {
+      headers: {
+        'User-Agent': 'Amazon CloudFront',
+        'Cloudfront-Is-Tablet-Viewer': 'true'
+      }
+    })
+    const { isMobile, isMobileOrTablet } = parseHtml(html)
+
+    expect({ isMobile, isMobileOrTablet }).toEqual({ isMobile: false, isMobileOrTablet: true })
+  })
+
+  it('detects cloudfront headers - desktop', async () => {
+    const html = await $fetch('/', {
+      headers: {
+        'User-Agent': 'Amazon CloudFront',
+        'Cloudfront-Is-Desktop-Viewer': 'true'
+      }
+    })
+    const { isMobile, isMobileOrTablet } = parseHtml(html)
+
+    expect({ isMobile, isMobileOrTablet }).toEqual({ isMobile: false, isMobileOrTablet: false })
```

Comment:
You should check for `isDesktop` here

Reply:
Updated ✅
You should consider updating Cloudeflare's desktop's test then (`detects cloudflare headers - desktop`) as I basically copied it 👍

Reply:
You're right, that's a copy and paste mistake I made while rewriting the tests. If you can, update it thanks!

I'd suggest to ping someone from the Nuxt team in order to get this PR noticed, as I don't have any permissions here.

Reply:
Cloudflare desktop test updated as well ✅ 

>I'd suggest to ping someone from the Nuxt team in order to get this PR noticed, as I don't have any permissions here.

ping @dotneet 👋 😊 

Apart from (hopefully) merging this, would it be possible to bring these changes to `v2.*.*`, considering everything's on `master` ?

Reply:
It should be possible, you would need to modify the [2.1.0](https://github.com/nuxt-community/device-module/tree/v2.1.0) branch.